README.md

```diff
@@ -34,38 +40,35 @@ The MSAL preview library for iOS and macOS gives your app the ability to begin u
         } 
 ```
 
-## App Registration 
-
-You will need to have a native client application registered with Microsoft using our [App Registration Portal](http://apps.dev.microsoft.com). You must do this even if you have previousdly registered your app with the legact portal. Once done, you will need add the redirect URI of `msal<your-client-id-here>://auth` in the portal.
-
-
-## Installation
+## Optional: Register your App  
 
-We use [Carthage](https://github.com/Carthage/Carthage) for package management during the preview period of MSAL. This package manager integrates very nicely with XCode while maintaining our ability to make changes to the library. The sample is set up to use Carthage.
+The app comes pre-configured for testing.  If you would like to register your own app, please follow 
+the steps below. 
 
-##### If you're building for iOS, tvOS, or watchOS
+You will need to have a native client application registered with Microsoft using the 
+[App Registration Portal](https://apps.dev.microsoft.com/portal/register-app?appType=mobileAndDesktopApp&appTech=android). 
 
-1. Install Carthage on your Mac using a download from their website or if using Homebrew `brew install carthage`.
-1. We have already created a `Cartfile` that lists the MSAL library for this project on Github. We use the `/dev` branch.
-1. Run `carthage update`. This will fetch dependencies into a `Carthage/Checkouts` folder, then build the MSAL library.
-1. On your application targets’ “General” settings tab, in the “Linked Frameworks and Libraries” section, drag and drop the `MSAL.framework` from the `Carthage/Build` folder on disk.
-1. On your application targets’ “Build Phases” settings tab, click the “+” icon and choose “New Run Script Phase”. Create a Run Script in which you specify your shell (ex: `/bin/sh`), add the following contents to the script area below the shell:
+To create an app,  
+1. Click the `Add an app` button inside the *Converged Apps* section.
```

Comment:
The *converged apps* section only appears if you have MSA apps (using the MSA SDK). otherwise it's hidden

Reply:
That was I get stealing from @danieldobalian. Fixed.